backend/schema/components/proxy-host-object.json

```diff
@@ -77,6 +79,16 @@
 			"example": true,
 			"type": "boolean"
 		},
+		"enable_proxy_protocol": {
+			"description": "Enable PROXY Protocol support",
+			"example": true,
+			"type": "boolean"
```

Comment:
CI is failing because of this.

```
[validateSwaggerSchema ERROR] Validation Errors: [
  {
    "instancePath": "/enable_proxy_protocol",
    "schemaPath": "#/properties/enable_proxy_protocol/
    "keyword": "type",
    "params": {
      "type": "boolean"
    },
    "message": "must be boolean",
    "schema": "boolean",
    "parentSchema": {
      "description": "Enable PROXY Protocol support",
      "example": true,
      "type": "boolean"
    },
    "data": 0
  }
]
```

Reply:
This is probably due to a missing part of the ORM glue. Add the `enable_proxy_protocol` string to the array of `boolFields` in

```
backend/models/proxy_host.js:14
```